cli/borges/consumer.go

```diff
@@ -31,6 +32,7 @@ func (c *consumerCmd) Execute(args []string) error {
 		core.ModelRepositoryStore(),
 		core.RootedTransactioner(),
 		borges.NewTemporaryCloner(core.TemporaryFilesystem()),
+		lock.NewLocal(),
```

Comment:
we should call here to the container to obtain a lock instance depending of the environment.